src/ImageSharp/Colors/Color.cs

```diff
@@ -112,6 +112,9 @@ public Color(uint packed)
             this.packedValue = packed;
         }
 
+        /// <inheritdoc />
+        public BulkPixelOperations<Color> BulkOperations => new BulkPixelOperations<Color>();
```

Comment:
As we are are newing something up, maybe make the Interface a method named `CreateBulkOperations()` instead of a property, makes people realise that it has something they shouldn't just dot into to use.

And the comment pointing people the the preferred accessor would also help.